rpm/sailfish-secrets.spec

```diff
@@ -213,6 +213,18 @@ Requires:   libsailfishcryptopluginapi = %{version}-%{release}
 %{summary}.
 
 
+%package -n sailfishsecretstool
+Summary:    Command line tool to interact with the Sailfish OS Secrets and Crypto service
+Group:      Applications/System
+BuildRequires:  pkgconfig(Qt5Core)
+BuildRequires:  pkgconfig(Qt5DBus)
+Requires:   libsailfishcrypto = %{version}-%{release}
+Requires:   libsailfishsecrets = %{version}-%{release}
```

Comment:
Can you use pkgconfig in Requires rather than BuildRequires?  My rpm knowledge is close to zero, but the .pc file probably comes from the -devel package which we don't want installed to the device.